### PR TITLE
Added and adjusted mochawesome as a development dependency

### PR DESCRIPTION
**Pull request by Diegocortes15**
image

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -194,3 +194,5 @@ $RECYCLE.BIN/
 
 # End of https://www.toptal.com/developers/gitignore/api/windows,visualstudiocode,node
 
+## Reports ##
+report
```
Comment (AlejaGonzalezV) [inline .gitignore:198]: Add an empty line at the end (EOF)

File: package.json [hunk 2]
```diff
@@ -4,7 +4,7 @@
 	"description": "This is a Workshop about Api Testing in JavaScript",
 	"main": "index.js",
 	"scripts": {
-		"test": "mocha -t 5000"
+		"test": "mocha -t 5000 --reporter mochawesome --reporter-options reportDir=report,reportFilename=ApiTesting"
```
Comment (AlejaGonzalezV) [inline package.json:7]: Please fix identation over all the file, for example:
```suggestion
	"test": "mocha -t 5000 --reporter mochawesome --reporter-options reportDir=report,reportFilename=ApiTesting"
```
When open a { use only one tab for identation

File: package.json [hunk 3]
```diff
@@ -4,7 +4,7 @@
 	"description": "This is a Workshop about Api Testing in JavaScript",
 	"main": "index.js",
 	"scripts": {
-		"test": "mocha -t 5000"
```
Comment (AlejaGonzalezV) [inline package.json:7]: Please don't delete the tab identation, just add one tab. Do the same for nested properties as type, chai, http-status-codes and so on

File: package.json [hunk 4]
```diff
@@ -4,7 +4,7 @@
...
+	"test": "mocha -t 5000 --reporter mochawesome --reporter-options reportDir=report,reportFilename=ApiTesting"
```
Comment (AlejaGonzalezV) [inline package.json:7]: Please don't remove the identation, just add one tab space. Do the same for other nested lines as: type, chai, http-status-codes and so on

**Review by holgiosalos — APPROVED** [review]

**Pull request merged**